### DISCUSSION TITLE
Freeze in browser

### DISCUSSION THREAD

**Discussion by TAGVL** [dwainscheeren/dwains-lovelace-dashboard#372] [Q&A]
Yesterday I configured the basics of Dwains Dashboard. On iOS everything I setup untill now seems to work, but in my browser (Chrome) when I open HA in Dwains it keeps freezing. 'Outside' of Dwains Dashboard everything works fine. I assume it has to do with something in a yaml file, but any idea where to start?

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
